pep-3156.txt

```diff
@@ -943,6 +970,52 @@ traceback.  In some cases they are caught and re-raised.  (Examples of
 this category include ``KeyboardInterrupt`` and ``SystemExit``; it is
 usually unwise to treat these the same as most other exceptions.)
 
+The event loop passes caught but not handled exceptions into own's
+*exception handler*.  The handler is a callback which accepts
+*context* dict as a parameter::
+
+   def exception_handler(context):
+       ...
+
+TBD: describe context's keys.
+
+The loop has the following methods related to exception handling:
+
+- ``get_exception_handler()`` returns the current handler registered
+  in the loop.
+
+- ``set_exception_handler(handler)`` assings new handler.
+
+- ``default_exception_handler(context)`` is *default* handler for the
+  loop implementation.
+
+- ``call_exception_handler(context)`` passes *context* into the
+  registered exception handler. The call allows to handle uncaught
+  exceptions uniformly by third-party libraries.
+
+  The loop uses ``default_exception_handler()`` if the default was not
+  overridden by explicit ``set_exception_handler()`` call.
+
+Debug Mode
+----------
+
+By default the loop operates in *release* mode for sake of performance.
+But application may enable *debug* mode.
+
+In this mode many additional checks are enabled, for example:
+
+- source tracebacks are available for unrolled exceptions in futures/tasks.
+
+- loop checks callback execution time for blaming blocking code.
```

Comment:
it's `loop.slow_callback_duration` attribute, callback's execution time is checked against the value in debug mode.
Perhaps I should describe the behavior in the PEP.